packages/plugin-search/test/integration/spec/search.js

```diff
@@ -0,0 +1,486 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import '../..';
+
+import CiscoSpark from '@ciscospark/spark-core';
+import {assert} from '@ciscospark/test-helper-chai';
+import testUsers from '@ciscospark/test-helper-test-users';
+import retry from '@ciscospark/test-helper-retry';
+
+import '@ciscospark/plugin-conversation';
+import '@ciscospark/plugin-wdm';
+
+import {map, countBy} from 'lodash';
+import uuid from 'uuid';
+import util from 'util';
+
+describe(`plugin-search`, () => {
+  describe(`#people`, () => {
+    let bot, checkov, mccoy, spark;
+
+    before(() => testUsers.create({count: 2})
+      .then((users) => {
+        [checkov, mccoy] = users;
+
+        spark = new CiscoSpark({
+          credentials: {
+            authorization: checkov.token
+          }
+        });
+
+        return spark.mercury.connect();
+      })
+      .then(() => spark.request({
+        api: `hydra`,
+        resource: `bots`,
+        method: `POST`,
+        body: {
+          displayName: `Personal Assistant`,
+          email: `${uuid.v4()}@sparkbot.io`
+        }
+      }))
+      .then((results) => {
+        bot = results.body;
+      })
+    );
+
+    after(() => {
+      spark.mercury.disconnect();
+    });
+
+    it(`searches for users with a specific string`, () => {
+      return spark.search.people({query: `abc`})
+        .then((users) => {
+          assert.isArray(users);
+        });
+    });
+
+    it(`retrieves a specific user`, () => {
+      return spark.search.people({query: mccoy.email})
+        .then((users) => {
+          assert.isAbove(users.length, 0, `At least one user was found`);
+          assert.equal(users[0].id, mccoy.id);
+        });
+    });
+
+    it(`retrieves a bot`, () => {
+      return spark.search.people({query: bot.email, includeRobots: true})
+        .then((bots) => {
+          assert.isAbove(bots.length, 0, `At least one bot was found`);
+          assert.equal(bots[0].email, bot.email);
+        });
+    });
+
+    it(`doesn't retrieve a bot because of parameter`, () => {
+      return spark.search.people({query: bot.email, includeRobots: false})
+        .then((bots) => {
+          assert.equal(bots.length, 0, `No bots are returned`);
+        });
+    });
```

Comment:
I think that's work that still underway in one of the message scrums, but now that Search#search just passes through the options passed to it; it should just work without sdk modifications. That said, the message scrum working on it *should* add sdk tests for the feature.